src/processors/knex-processor.ts

```diff
@@ -82,11 +98,38 @@ export default class KnexProcessor<
     return pluralize(type);
   }
 
-  filtersToKnex(filters: {}): {} {
+  filtersToKnex(builder, filters: {}) {
+    const processedFilters = [];
+
     Object.keys(filters).forEach(
-      key => filters[key] === undefined && delete filters[key]
+      key => filters[key] === undefined && delete filters[key],
     );
 
-    return filters;
+    Object.keys(filters).forEach((key) => {
+
+      let value = filters[key];
+      let operator = getOperator(filters[key]);
+
+      if (!operator) {
+        operator = '=';
+      }
+
+      if (value.substring(value.indexOf(':') + 1)) {
+        value = value.substring(value.indexOf(':') + 1)
+      }
+
+      value = value !== 'null' ? value : 0;
```

Comment:
@aklkv Why are we converting `null` to `0`?

Reply:
it come as a string 'null'